product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/1_6_1-synapse-configCompositeApplication/pom.xml

```diff
@@ -0,0 +1,159 @@
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.wso2.ei.product-scenarios.scenario_1_6_1</groupId>
+  <artifactId>1_6_1-synapse-configCompositeApplication</artifactId>
+  <version>1.0.0</version>
+  <packaging>carbon/application</packaging>
+  <name>1_6_1-synapse-configCompositeApplication</name>
+  <description>1_6_1-synapse-configCompositeApplication</description>
+  <properties>
+    <artifact.types>jaggery/app=zip,synapse/priority-executor=xml,synapse/inbound-endpoint=xml,service/rule=aar,synapse/message-store=xml,event/stream=json,service/meta=xml,datasource/datasource=xml,synapse/proxy-service=xml,bpel/workflow=zip,synapse/sequence=xml,synapse/endpointTemplate=xml,carbon/application=car,wso2/gadget=dar,synapse/api=xml,synapse/event-source=xml,synapse/message-processors=xml,event/receiver=xml,lib/dataservice/validator=jar,synapse/template=xml,synapse/endpoint=xml,lib/carbon/ui=jar,lib/synapse/mediator=jar,event/publisher=xml,synapse/local-entry=xml,synapse/task=xml,webapp/jaxws=war,registry/resource=zip,synapse/configuration=xml,service/axis2=aar,synapse/lib=zip,synapse/sequenceTemplate=xml,event/execution-plan=siddhiql,service/dataservice=dbs,web/application=war,lib/library/bundle=jar</artifact.types>
+    <org.wso2.ei.product-scenarios.scenario_01.sequence_._callPOXEndpointSeq>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.sequence_._callPOXEndpointSeq>
+    <org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_1_SoapToPoxMsgEnrichWithChildProxy>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_1_SoapToPoxMsgEnrichWithChildProxy>
+    <org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_SoapToPoxMsgEnrichWithChildProxy>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_SoapToPoxMsgEnrichWithChildProxy>
+    <org.wso2.ei.product-scenarios.scenario_01.sequence_._fullLogSeq>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.sequence_._fullLogSeq>
+    <org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_2_SoapToPoxEnrichWithXpathAsChildProxy>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_2_SoapToPoxEnrichWithXpathAsChildProxy>
+    <org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_1_Proxy_SoapToPoxMsgEnrichWithChild>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_1_Proxy_SoapToPoxMsgEnrichWithChild>
+    <org.wso2.ei.product-scenarios.scenario_01.endpoint_._POX_EP>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.endpoint_._POX_EP>
+    <org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_2_Proxy_SoapToPoxEnrichWithXpathAsChild>capp/EnterpriseServiceBus</org.wso2.ei.product-scenarios.scenario_01.proxy-service_._1_6_1_2_Proxy_SoapToPoxEnrichWithXpathAsChild>
```

Comment:
_1_6_1_2_SoapToPoxEnrichWithXpathAsChildProxy and _1_6_1_2_Proxy_SoapToPoxEnrichWithXpathAsChild both are there

Reply:
Corrected.